src/CircularProgress.js

```diff
@@ -33,10 +26,10 @@ export default class CircularProgress extends React.Component {
 
   render() {
     const { size, width, tintColor, backgroundColor, style, rotation, linecap, children } = this.props;
-    const backgroundPath = this.circlePath(size / 2, size / 2, size / 2 - width / 2, 0, 360);
+    const backgroundPath = this.circlePath(size / 2, size / 2, size / 2 - width / 2, 0, 360 * 99.9 / 100);
```

Comment:
@andreiciceu Makes sense. Is this only a problem on Android, though? Question is whether we should do `.999` on Android and `360` on iOS. I think doing the same thing on both platforms, as with your PR, is the best approach. That way the library performs in the same way across platforms and is easier to reason about.

Good job with the PR 👍